app-backend/build.sbt

```diff
@@ -145,13 +144,14 @@ lazy val noPublishSettings = Seq(
 )
 
 lazy val publishSettings = Seq(
-  // Add the default sonatype repository setting
-  publishTo := sonatypePublishTo.value,
-  publishMavenStyle := true,
   organization := "com.rasterfoundry",
   organizationName := "Raster Foundry",
-  organizationHomepage := Some(new URL("https://www.rasterfoundry.com")),
+  organizationHomepage := Some(new URL("https://rasterfoundry.com/")),
```

Comment:
If we are going to change the URL, the most up-to-date URL is: https://rasterfoundry.azavea.com/.